### PR TITLE
Add step by step ETL instructions

### PR DESCRIPTION
**Pull request by buckley-w-david**
This PR adds a step-by-step series of instructions on adding an example element to the ETL process (AIR50P)

### CODE REVIEW

**Comment by buckley-w-david** [comment]
A nice rendered document can be viewed at https://github.com/cds-snc/nrcan_api/blob/76fb05a52a19452b555887a8621369912d035848/resource/add_to_etl.md

File: resource/add_to_etl.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+```
+
+Commit and push your changes, and make a Pull Request
+
+## Adding to Transform
+
+### `nrcan_etl/src/energuide/dwelling.py`
+1. Add a new property to `_ParsedDwellingDataRow` for the air leakage
+
+This class is a container for all the fields, both dwelling and evaluation level, that the input data is parsed into at the beginning so the rest of the pipeline has a dependable known structure to use.
```
Comment (evadb) [inline resource/add_to_etl.md]: I'd break apart this statement for clarity. Something like this maybe: "This class is a container for all the fields at both the dwelling and evaluation levels. The input data is parsed into this container at the beginning of the process so the rest of the pipeline has a dependable known structure to use."

File: resource/add_to_etl.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+```
+class _ParsedDwellingDataRow(typing.NamedTuple):
+    ...
+    air_leakage: measurement.Measurement
+    ...
+
+```
+
+2. Add new elements to the `_SCHEMA` directionary in the `Dwelling` class
```
Comment (evadb) [inline resource/add_to_etl.md]: missing a period after class

**Comment by buckley-w-david** [comment]
ping @XavierPhanel @vadjr You guys can follow these steps as an outline for how to add a field to the ETL

**Pull request merged**